dist/index.html

```diff
@@ -12,7 +12,8 @@
     ></script>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>WebDevII-Final-Project</title>
-    <script src="dist/main.js" defer></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"> </script>
```

Comment:
It would be safer to install using npm instead of cdn, I think.
[reason]
It is better to keep the scope of variables as narrow as possible to make the code less buggy